MEMBER_TRAVEL_FUND.MD

```diff
@@ -0,0 +1,51 @@
+Foundation to travel and spread knowledge about and support the foundation
+and the Node.js Foundation projects.
+
+## Restrictions
+
+* Candidates must be an indvidual member of the Node.js Foundation.
+
+## Process
+
+Any member can apply for travel funds. Each request may take up to a month
+for the Admin members to approve or decline.
+
+### Request
+* Open a pull request against this repository which adds an entry to the table at the bottom of this file.
+ * Include the name of the event you plan to attend.
+ * Include the location of the event and where you will be traveling from.
+ * Include the presentation you intend to give, if applicable.
+ * Include the size of the stipend you wish to receive.
+  * Reimbursement stipends are expected to vary with travel distance.
+  * In the PR description, tag the group relevant to the request: @nodejs/tsc or @nodejs/commcomm-members. 
+  If unsure, just pick one, and that group will tag with the other group if they believe it is more appropriate.
+* Once the final amount spent is known, update the table again with that information.
+
+### Reimbursement
+  
+Once the request has been approved, provide receipts as attachments in an email stating your name, the participation covered, and the total approved for reimbursement. Send to ap@linuxfoundation.org and cc tracyhinds@linuxfoundation.org with subject `Node.js Member Travel Fund`. Due to privacy, the Individual Members list is not public. This team will verify the requester is on the Individual Members list before funds are dispersed. This dispersement is generally processed within 30 days. The community manager or a member of the Node.js Foundation team within the Linux Foundation will report back amounts consumed from the travel allocation on a monthly basis to the Chairs.
+
+**The following considerations are made for approval of the request:**
+
+* Impact: Preference given to underserved communities. More specifically,
+within a subject the foundation is trying to promote awareness about (e.g.
+general knowledge about the foundation) there are people, communities and
+geographies where that knowledge is not very widespread. This is where
+preference will be given. For instance, an event in San Francisco would be
+less attractive than an event of the same size in Kansas City.
+* Cost: The larger the stipend the more critically the travel fund admins will 
+consider the application. Budget for this program is a finite resource.
+* Equity: Preference given to individuals who do not have a corporate travel fund and have
+not previously received a stipend.
+
+Examples of prior travel requests include attending a TC39 meeting, Collab Summit, and Code + Learn mentoring.
+
+Name | Event | Kind of participation | Location | Date | Stipend size
+---- | ----- | --------------------- | -------- | ---- | ------------
+Example Person | NINA 2017 | Collaborator Summit and Code & Learn | Vancouver, BC, CA | 4 Oct – 7 Oct 2017 | US$ 2000
```

Comment:
should we include the allocations that are already listed in the TSC version of this doc?

Reply:
it may be desirable for bookeeping

Reply:
I agree we should just copy the existing data over to here so that the info is all in one place.  We can then probably remove the copy in the TSC repo to avoid confusion.

Reply:
will do. Good catch.